mooseutils/__init__.py

```diff
@@ -0,0 +1,10 @@
+#* This file is part of MOOSETOOLS repository
+#* https://www.github.com/idaholab/moosetools
+#*
+#* All rights reserved, see COPYRIGHT for full restrictions
+#* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+#*
+#* Licensed under LGPL 2.1, please see LICENSE for details
+#* https://www.gnu.org/licenses/lgpl-2.1.html
+
+from .validate import validate_extension, validate_files_exist
```

Comment:
You did this part. You can just call these from mooseutils directly. See comment above.